src/assets/styles/main.scss

```diff
@@ -89,3 +89,182 @@ img {
     background: $mainBg;
     padding-bottom: 30px;
 }
+
+.custom-field {
+    margin-bottom: 10px;
+    padding-bottom: 20px;
+    position: relative;
+}
+
+.custom-label {
+    display: block;
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 20px;
+    letter-spacing: 0.888889px;
+    text-transform: uppercase;
+    color: $primary;
+    mix-blend-mode: normal;
+    opacity: 0.8;
+    margin-bottom: 13px;
+}
+
+.custom-checkbox {
+    &__input {
+        display: none;
+        &:checked + .custom-checkbox__label-wrap {
+            .custom-checkbox__box {
```

Comment:
nit: avoid deep nesting like this. It's hard to read and creates redundant selector weight
scss styleguide recommends 3 level max